docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_managed_identity/linux/_index.md

```diff
@@ -0,0 +1,405 @@
+---
+type: docs
+title: "Ubuntu"
+linkTitle: "Ubuntu"
+weight: 1
+description: >
+---
+
+## Use Managed Identity on an Azure Arc-enabled server
```

Comment:
```suggestion
## Use Managed Identity on an Azure Arc-enabled servers
```

Reply:
Does this work?

Reply:
Perhaps the following? I can't see a way to suggest changes to a change, so I'll commit yours and then do a follow on commit on line 9?

```suggestion
## Using Managed Identity on Azure Arc-enabled servers
```